oidc/identity.go

```diff
@@ -42,3 +43,15 @@ func IdentityFromClaims(claims jose.Claims) (*Identity, error) {
 
 	return &ident, nil
 }
+
+// CopyAdditonalClaims copies all additional claims to claims
+func (ident *Identity) CopyAdditonalClaims(claims jose.Claims) {
```

Comment:
Can you elaborate more on why you want this to be a method we expose as part of the package API? It seems unnecessary.